### DISCUSSION TITLE
Allow single color background

### DISCUSSION THREAD

**Discussion by jaskij** [linuxserver/Heimdall#994] [Ideas]
As in the title - I'd like the option to, instead of uploading an image, set a single color background via a color picker or something similar.

Why? The default background is too busy for me, otherwise I'd leave it at whatever comes stock.

**Comment by Efus10n** [comment]
@jaskij I also like the idea for more background options such as color, image tiling yes/no. The current version 2.4.11 (I don't know when it was added) you can use a workaround in the form of a line of custom CSS. 

In the settings, go to the Advanced section and edit the Custom CSS. Putting in this line will supercede the default background image. Use whatever color hex or HTML color name you like.
`#app .content {background-color: #001a33;}`
